packetbeat/tests/system/test_0062_cassandra.py

```diff
@@ -23,7 +23,21 @@ def test_create_keyspace(self):
         o = objs[0]
 
         assert o["type"] == "cassandra"
+        assert o["event.dataset"] == "cassandra"
+        assert o["event.duration"] == 62453000
+        assert o["event.start"] == o["@timestamp"]
+        assert o["event.end"] == "2016-06-28T09:03:53.502Z"
+        assert o["client.ip"] == "127.0.0.1"
+        assert o["client.port"] == 52749
+        assert o["client.bytes"] == 133
+        assert o["server.ip"] == "127.0.0.1"
         assert o["server.port"] == 9042
+        assert o["server.bytes"] == 44
+        assert o["network.bytes"] == 133 + 44
+        assert o["network.type"] == "ipv4"
+        assert o["network.transport"] == "tcp"
+        assert o["network.protocol"] == "cassandra"
+        assert o["network.community_id"] == "1:bCORHZnGIk6GWYaE3Kn0DOpQCKE="
```

Comment:
It's not obvious to me here, but are you also populating `source.[ip|port]` and `destination.[ip|port]`? If so, could you add tests for that?

In ECS we recommend always filling source and destination, and also filling client/server when appropriate. But the source/destination pair should always be there.

Reply:
All of these updated "datasets" automatically populate the client/server with the source/destination if they have not been populated by the code.

https://github.com/elastic/beats/blob/ce4f474676c28bf6d075b899ef7c436ac2bd180a/packetbeat/pb/event.go#L221-L231

I will be updating the python test at the end (after all of packetbeat is updated) to check for the existence of all of the common fields like source/destination/client/server/network.transport/event.dataset/event.start/etc.